cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -186,12 +191,19 @@ public void setVoltageAngleNodeBreaker() {
                 return;
             }
             LOG.warn(
-                    "Can't find a calculated Bus to set Voltage, Angle, but found a configured Bus {}. Connectivity node {}",
-                    bus, id);
+                "Can't find a calculated Bus to set Voltage, Angle, but found a configured Bus {}. Connectivity node {}",
+                bus, id);
         }
         setVoltageAngle(bus);
     }
 
+    private boolean iidmNodeIsValid(VoltageLevel.NodeBreakerView topo, int iidmNode) {
+        // A node is valid only if it has some equipment attached
+        // The list of valid nodes is obtained from the node-breaker view
+        // We assume the returned list is sorted
```

Comment:
Not sure we should assume that, or we should update the documentation.
Anyway, is it the best way to check if a terminal is associated to this node: maybe we should add a convenient method to get an `Optional<Terminal>` from a node?

Reply:
It is not enough because a node is still valid if an internal connection or a switch is attached to it (with no terminal)... I added a convenient method to check if there is any equipment attached to the node at NodeBreakerView level to delete this method.